mmv1/products/blockchainnodeengine/BlockchainNodes.yaml

```diff
@@ -0,0 +1,210 @@
+
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/blockchain-node-engine'
+  api: 'https://cloud.google.com/blockchain-node-engine/docs/reference/rest/v1/projects.locations.blockchainNodes'
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'blockchain_nodes_basic'
+    primary_resource_id: 'default_node'
+    vars:
+      blockchain_node_id: 'blockchain_basic_node'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'blockchain_nodes_geth_details'
+    primary_resource_id: 'default_node_geth'
+    vars:
+      blockchain_node_id: 'blockchain_geth_node'
+parameters:
+  - !ruby/object:Api::Type::String
+    name: location
+    description: |
+      Location of Blockchain Node being created.
+    immutable: true
+    required: true
+    url_param_only: true
+  - !ruby/object:Api::Type::String
+    name: blockchainNodeId
+    url_param_only: true
+    required: true
+    description: |
+      ID of the requesting object.
+properties:
+  - !ruby/object:Api::Type::String
+    name: name
+    output: true
+    description: |
+      The number of the google project that firebase is enabled on.
+  - !ruby/object:Api::Type::String
+    name: createTime
+    output: true
+    description: |
+      The timestamp at which the blockchain node was first created.
+  - !ruby/object:Api::Type::String
+    name: updateTime
+    output: true
+    description: |
+      The timestamp at which the blockchain node was last updated.
+  - !ruby/object:Api::Type::KeyValueLabels
+    name: labels
+    description: |
+      User-provided key-value pairs
+  - !ruby/object:Api::Type::NestedObject
+    name: connectionInfo
+    output: true
+    description: |
+      The connection information through which to interact with a blockchain node.
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: endpointInfo
+        output: true
+        description: |
+          The endpoint information through which to interact with a blockchain node.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: jsonRpcApiEndpoint
+            output: true
+            description: |
+              The assigned URL for the node JSON-RPC API endpoint.
+          - !ruby/object:Api::Type::String
+            name: websocketsApiEndpoint
+            output: true
+            description: |
+              The assigned URL for the node WebSockets API endpoint.
+  - !ruby/object:Api::Type::NestedObject
+    name: ethereumDetails
+    description: |
+      User-provided key-value pairs
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: gethDetails
+        description: |
+          User-provided key-value pairs
+        properties:
+          - !ruby/object:Api::Type::Enum
+            name: garbageCollectionMode
+            immutable: true
+            values:
+              - :GARBAGE_COLLECTION_MODE_UNSPECIFIED
```

Comment:
Is this a valid value for users to send in API requests?

Reply:
yeah, not really needed for users to use.